crates/ra_ide/Cargo.toml

```diff
@@ -17,6 +17,12 @@ indexmap = "1.3.2"
 itertools = "0.9.0"
 log = "0.4.8"
 rustc-hash = "1.1.0"
+rand = { version = "0.7.3", features = ["small_rng"] }
+url = "*"
+maplit = "*"
```

Comment:
We don't use helper crates: https://github.com/rust-analyzer/rust-analyzer/pull/5625/commits/389ad925ffc534e46a2374030b0fbb3d4f75f3c9

Reply:
I've gotten rid of `maplit`, does `url` also fall under this umbrella?

Reply:
url should be fine, for now. Long term we should replace it with something which doesn't do URL parsing and just losslessly transfers strings

Reply:
The said long-term is blocked on https://github.com/microsoft/language-server-protocol/issues/67 :-(